src/Data/List/Relation/Unary/All/Properties.agda

```diff
@@ -11,12 +11,13 @@ module Data.List.Relation.Unary.All.Properties where
 open import Axiom.Extensionality.Propositional using (Extensionality)
 open import Data.Bool.Base using (Bool; T; true; false)
 open import Data.Bool.Properties using (T-∧)
-open import Data.Empty
+open import Data.Empty using (⊥-elim)
```

Comment:
Ugh! Simultaneous import of `Data.Empty` (for `⊥-elim`) and `Relation.Nullary.Negation.Core` for `contradiction`... needs fixing at some point.